common/core/events.h

```diff
@@ -26,30 +26,15 @@
 /*****************************************************************************
  * MACROS AND DEFINES
  *****************************************************************************/
-#define EVT_CONFIG_UI_EVT (1 << 0)
-#define EVT_CONFIG_USB_EVT (1 << 1)
-#define EVT_CONGIG_NFC_EVT (1 << 2)
+#define EVT_CONFIG_UI (1 << 0)
+#define EVT_CONFIG_USB (1 << 1)
+#define EVT_CONFIG_NFC (1 << 2)
```

Comment:
Just because you've already touched on this, I suggest we avoid the acronym evt and use full name event.

Reply:
ATCG

Reply:
changed